api/turing/api/request/request_test.go

```diff
@@ -65,7 +65,7 @@ func makeTuringExperimentConfig(clientPasskey string) json.RawMessage {
 	return expEngineConfig
 }
 
-var defaultRouteID string = "default"
+var defaultRouteID = "default"
```

Comment:
Removed the redundant type declared after the variable name. I've repeated this a couple of times across the codebase to address prompts from our linters.